### ISSUE TITLE
[Question]: Is it possible to get result from single-value request

### ISSUE THREAD

**Issue by isavitsky** [swoga/mikrotik-exporter#66]
Hello,

I've been struggling for a couple of days now for getting a sensible result from single-value Mikrotik API requests.
For example, this one:
```
/ip/dhcp-server/lease/print
=active=
=count-only=
?server=LAN-8-floor

<<< /ip/dhcp-server/lease/print
<<< =active=
<<< =count-only=
<<< ?server=LAN-8-floor
<<< 
>>> !done
>>> =ret=7
>>>
```
As we can see in this example API returns an unnamed/anonymous object that doesn't fall into `re={…}` response:
```
{ret: 7}
```
In the debug session we see no response to such requests.
The immediate trace reply is:
```
TRC response channel closed
```

My configuration is as follows:
```
modules:
  - name: testx
    commands:
      - command: |-
          /ip/dhcp-server/lease/print
          ?server=LAN-8-floor
          =active=
          =count-only=
        metrics:
          - metric_name: dhcp_entries
            param_name: ret
            param_type: int
            metric_type: gauge
            labels:
              - param_name: source
                param_type: string
                value: dhcp
```
So, is it possible somehow to get this value?

**Comment by swoga** [comment]
Hi,

good idea, I have implemented this feature.
Works exactly like you tried, with `param_name: ret`.

I haven't created a release yet, so for testing with Docker use the `mikrotik-exporter:dev` image.
Let me know if everything works, then I'll create a release!

**Comment by isavitsky** [comment]
Hi,
Thanks for the quick update. I've tested your new `:dev` image with my configuration and it is working good.

One note, though. I still had to rebuild my own image from your source applying the patch from issue #54 .
Otherwise container logs are being spammed with lines and lines of time parse errors:
```
swoga-exporter      | 2024-02-29T12:33:18Z ERR failed to parse datetime error="parsing time \"2024-02-29 14:17:52\": month out of range" command_no=1 connection_no=0 metric_name=last_link_up module=interface request_no=413 sentence_no=13 target=router-traffic word="2024-02-29 14:17:52"
swoga-exporter      | 2024-02-29T12:33:18Z ERR failed to parse datetime error="parsing time \"2024-02-29 02:49:16\": month out of range" command_no=1 connection_no=0 metric_name=last_link_down module=interface request_no=413 sentence_no=19 target=router-traffic word="2024-02-29 02:49:16"
```

**Comment by swoga** [comment]
Hi,

thanks for the note!
Somehow I didn't notice this during testing and we don't have v7 in production yet.
Have pushed a fix please try again with the `dev` image!

**Comment by swoga** [comment]
New release with both fixes is available! [v0.8.0](https://github.com/swoga/mikrotik-exporter/releases/tag/v0.8.0)
